### ISSUE TITLE
Missing Documentation for Weight Norm Function

### ISSUE THREAD

**Issue by schelv** [pytorch/pytorch#113657]
## 📚 Missing Documentation for Weight Norm Function

I noticed there's an advisory on the documentation page for the old weight norm function, directing users to the new function:
https://pytorch.org/docs/2.1/generated/torch.nn.utils.weight_norm.html

However, there doesn't seem to be documentation available for the new function. Ideally, I'd expect to find it here:
https://pytorch.org/docs/2.1/generated/torch.nn.utils.parametrizations.weight_norm.html

For reference, other parametrizations are well-documented:
- [Spectral Norm](https://pytorch.org/docs/2.1/generated/torch.nn.utils.parametrizations.spectral_norm.html)
- [Orthogonal](https://pytorch.org/docs/2.1/generated/torch.nn.utils.parametrizations.orthogonal.html)

Could we get documentation for the new weight norm function added? Thanks!

cc @svekars @carljparker @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

**Comment by malfet** [comment]
Transferred to pytorch, adding appropriate labels

**Comment by albanD** [comment]
This was fixed in https://github.com/pytorch/pytorch/pull/113783 today
Thanks for the report
